app/Http/Requests/PostRequest.php

```diff
@@ -26,9 +26,9 @@ public function rules()
             'campaign_id' => 'required|int',
             'campaign_run_id' => 'required|int',
             'caption' => 'string',
-            'status' => 'pending',
+            'status' => 'in:pending,accepted,rejected',
             'source' => 'string',
-            'remote_addr' => 'ip',
+            'remote_addr' => 'string',
```

Comment:
@sbsmith86 it still works with just one IP address, so I think it is always a string